Prototope/Behaviors.swift

```diff
@@ -0,0 +1,218 @@
+//
+//  Collisions.swift
+//  Prototope
+//
+//  Created by Saniul Ahmed on 07/02/2015.
+//  Copyright (c) 2015 Khan Academy. All rights reserved.
+//
+
+import Foundation
+
+//MARK: Public Behavior Types
+
+/** Protocol for types describing the "configuration" of a behavior. */
+public protocol BehaviorType {}
+
+/** Describes a behavior that just calls a closure/block on every heartbeat,
+    passing the host layer to the closure. */
+public struct BlockBehavior: BehaviorType {
```

Comment:
Got some feedback from the designers: since "blocks" aren't a thing in JS, this naming was a bit surprising. How about `ActionBehavior`?